### DISCUSSION TITLE
Icon Request: fa-bikini

### DISCUSSION THREAD

**Discussion by JWTechSolutions** [FortAwesome/Font-Awesome#21247] [Icon Requests]
I would like a two piece bikini icon. I like the first picture style more as it can also be used as a lingerie icon which would make it better for multiple uses. I want it to be vertical though like the second picture.

![blue-sexy-2-piece-bikini-ruffle-halter-bandeau-swimwear](https://cloud.githubusercontent.com/assets/9329531/5016547/0539a934-6a5c-11e4-89b4-4fbb888d76b4.jpg)

![41tt0buwy0l _sy445_](https://cloud.githubusercontent.com/assets/9329531/5016553/10660d66-6a5c-11e4-91b2-85e23b1e0c8f.jpg)

**Comment by ruvenss** [comment]
:+1:

**Comment by ddavid67** [comment]
+1

**Comment by aethior** [comment]
*Related Unicode Emoji* #222
👙 [U+1F459](https://emojipedia.org/emoji/%F0%9F%91%99/) Bikini

**Comment by sensibleworld** [comment]
This is a quick update to let folks know that we’re refining our icon request workflow behind the scenes, and as part of that we’re going through all open requests and filing them into several new buckets to be addressed.

In this case we’ve added this issue to our new list to be created! We can’t promise any firm timelines right now, but this will help us keep track of it.

We’ll update this issue when the icon is released into the wild 🙂
